src/server/search/search_family.cc

```diff
@@ -133,17 +157,21 @@ void SearchFamily::FtSearch(CmdArgList args, ConnectionContext* cntx) {
   string_view index_name = ArgS(args, 0);
   string_view query_str = ArgS(args, 1);
 
+  auto params = ParseSearchParamsOrReply(args.subspan(2), cntx);
+  if (!params)
```

Comment:
Nit: I'd prefer `if (!params.has_value())`, especially if you're using `auto` here as it makes it even clearer what `params` is